extract/extract/extractors/zalando.py

```diff
@@ -94,10 +94,14 @@ def extract_zalando(
     if price := first_offer.get("price", None):
         price = float(price)
 
-    sustainability_labels = get_sustainability(
-        parsed_page.beautiful_soup, label_mapping=label_mapping
+    unmapped_labels = set(get_sustainability(parsed_page.beautiful_soup))
+    sustainability_labels = sorted(
+        {label_mapping.get(label, CertificateType.UNKNOWN) for label in unmapped_labels}
     )
 
+    if unknown_labels := {label for label in unmapped_labels if label not in label_mapping}:
+        logger.info(f'unknown sustainability labels: {", ".join(map(repr, unknown_labels))}')
```

Comment:
Do you think this logging is important? - If not, we could extend `get_sustainability` a little bit to return a list of `sustainability_labels` as before. If yes, we should rename the function and some of the variables like `unmapped_labels`